presto-hive/src/main/java/com/facebook/presto/hive/metastore/SemiTransactionalHiveMetastore.java

```diff
@@ -2005,6 +2005,27 @@ public PartitionStatistics getStatisticsUpdate()
             return statisticsUpdate;
         }
 
+        public Table getAugmentTableForInTransactionRead()
```

Comment:
nit: either call this getTableForInTransactionRead() or augmentTableForInTransactionRead()

Reply:
Similar method for partition is called `getAugmentedPartitionForInTransactionRead`. Let's call it the same way, `getAugmentedTableForInTransactionRead`. I overlooked `Augment/Augmented`.

Reply:
ah yeah that's fine too.

Reply:
Thanks! Name fixed.